src/styles/main.scss

```diff
@@ -0,0 +1,70 @@
+.watch {
+  position: relative;
+  width: 500px;
+  height: 500px;
```

Comment:
It would be great to create variables for all main values
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/909e59e9-a326-4c5a-8b7f-b104e24e05bf)